TransIT.DAL/Models/DTOs/TokenDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace TransIT.DAL.Models.DTOs
+{
+    public class TokenDTO
+    {
+        public int Id { get; set; }
+        public string RefreshToken { get; set; }
+        public string AccessToken { get; set; }
+        public DateTime? CreateDate { get; set; }
+        public DateTime? ModDate { get; set; }
```

Comment:
Do we need date here?

Reply:
We do need, to know when token is expired.

Reply:
This is already encrypted in `AccessToken`
Moreover, you display only creation date, not expiration one